rulebook.tex

```diff
@@ -2041,12 +2043,42 @@ \subsection{Wifi Regulations}
 %%%
 
 
-\section{Tournament --- Main Track}
+\section{Tournament Setup}
+A tournament in the \ac{RCLL} is divided into two different tracks.
+While in the challenge track (see \refsec{sec:tournament-challenges}) teams
+need to complete challenges without direct opposition in order to accumulate
+points, the main track consists of matches, where two teams directly compete
+against each other in a full production scenario.
+
+Each tournament starts with a qualification day, where teams can attempt
+the production challenge (see \refsec{sec:challenge-cx}) from the challenge
+track in order to prove that they have the required prerequisites to participate
+in the main track. The schedule for each team on the qualification day will be
+provided by the \ac{OC} at the beginning of the tournament.
+
+Teams that complete the production challenge can chose the track that they
+want to participate in, the other teams will compete in the challenge track.
```

Comment:
Isn't the production challange the hardest challange? I guess once a team is able to finish this at he beginning they should play in the main track.

Reply:
i would think so too, but the content of this PR only implemented the stuff discussed in our meetings. Since we only talked about having the choice to participate in the main challenge i did not want to change it yet. Also, i can see that if a team just barely manages to build a C0, then it might want to remain in the challenge track until it is confident to build bigger products.